*** The reference

--- Yours

***************

*** 1,21 ****

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! hello, world
! printed via dynamic dispatch
! tested static dispatch and 'new IO'
! an int: 3
! gonna make a Blah
! gonna call doThemAll
! basics:
! more advanced:
! bigIf
! bigMath
! bigAssign
! bigWhile
! bigSelf
! bigStrCompare
! bigCase
! bigAttrAccess
! looks ok!
! COOL program successfully executed
  
--- 1,5 ----

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/bigexample.cl.my.s line 2051 column 9: Symbol "SELF_TYPE_protObj" not found in symbol table.
  
+ Processing terminated due to errors.
