*** The reference

--- Yours

***************

*** 1,15 ****

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Applying closure 0
! Applying closure 1
! Applying closure 17
! Applying closure 2
! Applying closure 15
! Applying closure 16
! Applying closure 4
! Applying closure 5
! Applying closure 12
! Applying closure 6
! Applying closure 7
! COOL program successfully executed
  
--- 1,82 ----

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 1094 column 9: "Main.main" declared global label but not defined.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 671 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 672 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 673 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 674 column 8: Symbol "Closure.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 679 column 8: Symbol "Main.main" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 689 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 690 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 691 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 692 column 8: Symbol "Closure8.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 702 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 703 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 704 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 705 column 8: Symbol "Closure7.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 715 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 716 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 717 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 718 column 8: Symbol "Closure9.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 728 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 729 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 730 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 731 column 8: Symbol "Closure6.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 741 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 742 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 743 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 744 column 8: Symbol "Closure5.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 754 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 755 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 756 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 757 column 8: Symbol "Closure10.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 767 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 768 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 769 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 770 column 8: Symbol "Closure11.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 780 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 781 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 782 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 783 column 8: Symbol "Closure4.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 793 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 794 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 795 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 796 column 8: Symbol "Closure12.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 806 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 807 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 808 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 809 column 8: Symbol "Closure3.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 819 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 820 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 821 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 822 column 8: Symbol "Closure13.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 832 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 833 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 834 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 835 column 8: Symbol "Closure14.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 845 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 846 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 847 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 848 column 8: Symbol "Closure2.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 858 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 859 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 860 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 861 column 8: Symbol "Closure15.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 871 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 872 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 873 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 874 column 8: Symbol "Closure16.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 884 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 885 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 886 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 887 column 8: Symbol "Closure1.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 897 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 898 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 899 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 900 column 8: Symbol "Closure17.apply" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 910 column 8: Symbol "Closure.get_parent" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 911 column 8: Symbol "Closure.get_x" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 912 column 8: Symbol "Closure.init" not found in symbol table.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/c.cl.my.s line 913 column 8: Symbol "Closure0.apply" not found in symbol table.
  
+ Processing terminated due to errors.
