*** The reference

--- Yours

***************

*** 1,4 ****

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! ./codegen-test-files//dispatch-void-static.cl:15: Dispatch to void.
  
--- 1,5 ----

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/dispatch-void-static.cl.my.s line 431 column 2: "lw": Too few or incorrectly formatted operands. Expected: lw $t1,-100($t2)
  
+ Processing terminated due to errors.
