*** The reference

--- Yours

***************

*** 1,4 ****

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! 17141611714163171416511714161171416317141653117141611714163171416511714161171416317141653171416117141631714165171416COOL program successfully executed
  
--- 1,5 ----

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/hairyscary.cl.my.s line 706 column 2: "lw": Too few or incorrectly formatted operands. Expected: lw $t1,-100($t2)
  
+ Processing terminated due to errors.
