*** The reference

--- Yours

***************

*** 1,4 ****

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! COOL program successfully executed
  
--- 1,6 ----

  MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar
  
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/objectequality.cl.my.s line 350 column 9: "Main.main" declared global label but not defined.
! Error in /Users/Kong/Library/Mobile Documents/com~apple~CloudDocs/Spring_2016/164/cs164/./codegen-test-files/objectequality.cl.my.s line 295 column 8: Symbol "Main.main" not found in symbol table.
  
+ Processing terminated due to errors.
